### DISCUSSION TITLE
Display brand description (normal and short) on brand page

### DISCUSSION THREAD

**Discussion by ECC64** [PrestaShop/PrestaShop#34588] [Ideas]
### The problem

The brand description (normal and short) don't displayed on brand page

### The solution

insert this lines :
 {block name='brand_description'}
        <div id="manufacturer-short_description">{$manufacturer.short_description nofilter}</div>
        <div id="manufacturer-description">{$manufacturer.description nofilter}</div>
  {/block}
in brands.tpl line 30

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by SharakPL** [comment]
It already is displayed in both classic-theme and new hummingbird theme. If anything short description should be removed from the brand page if full description is already there.

**Comment by kpodemski** [comment]
Hello @ECC64 

I think this problem might be related to the theme you use, in the default theme the description is well displayed.
